### ISSUE TITLE
model.eval() mode

### ISSUE THREAD

**Issue by giorgiobini** [uci-cbcl/UFold#18]
Hello,

I see you have BatchNorm2d layers, so the output of the model with change accordingly to the train() and eval() mode you use. I was wondering why you haven't tested the model with the eval() mode.

Thank you so much in advance!

**Comment by sperfu** [comment]
Hi,

We trained/evluated our model with batch size 1. Different sequences have very different running statistics, so we don't want the model to use the previously stored statistics.
